test/mouse_test.py

```diff
@@ -61,9 +61,65 @@ def todo_test_get_cursor(self):
         """Ensures get_cursor works correctly."""
         self.fail()
 
-    def todo_test_set_cursor(self):
+    def test_set_cursor(self):
         """Ensures set_cursor works correctly."""
-        self.fail()
+        size = (8, 8)
+        hotspot = (0, 0)
+        xormask = (0, 126, 64, 64, 32, 16, 0, 0)
+        andmask = (254, 255, 254, 112, 56, 28, 12, 0)
+        expected_cursor = (size, hotspot, xormask, andmask)
+
+        # Error should be raised when the display is uninitialized
+        with self.assertRaises(pygame.error):
+            pygame.display.quit()
+            pygame.mouse.set_cursor(size, hotspot, xormask, andmask)
+
+        pygame.display.init()
+        
+        try:
```

Comment:
Why are these asserts the tests in this `try` block?

Reply:
I put the tests in the try block for when there isn't a cursor thinking it should be caught for all the tests, but now looking back at the code it was unnecessary since the errors I was testing would have been raised before reaching the check for the cursor.